### ISSUE TITLE
Concept re-mapping is problematic for updates from concepticon

### ISSUE THREAD

**Issue by LinguList** [lexibank/zgraggenmadang#5]
the long dictionary of concept re-mappings will make it difficult to deal with the data.

We may have a problem here due to:

1. the original data of Madang in the `raw` folder being represented in a different way due to digitalization differences (and potential errors)
2. the original data in the `raw` folder being a merger of the four different books by Zgraggen. 

@SimonGreenhill, can you provide some info, how the concept list was created? 

@tresoldi, as a maintainer, can you please also have a look at this, and see how we can avoid the long list of remappings?

**Comment by SimonGreenhill** [comment]
i'll look into this next week. Think the issue is that the 4 volumes have different concepts.

**Comment by SimonGreenhill** [comment]
Yes, the original data is a merger of the 4 books, but these have a very high degree of overlap. What's the best way forward? do we keep this list for the dataset as-is and add the 4 zgraggen volumes into concepticon as new lists?
